scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt

```diff
@@ -80,10 +80,9 @@ class ScannerIntegrationFunTest : StringSpec() {
                 )
             )
 
-            val ortResult = scanner.scan(analyzerResultFile.readValue(), skipExcluded = false, emptyMap())
-            val result = yamlMapper.writeValueAsString(ortResult)
+            val result = scanner.scan(analyzerResultFile.readValue(), skipExcluded = false, emptyMap())
 
-            patchActualResult(result, patchStartAndEndTime = true) shouldBe expectedResult
+            patchActualResult(result.toYaml(), patchStartAndEndTime = true) shouldBe expectedResult
```

Comment:
Note that there's more test code that calls `yamlMapper.writeValueAsString(...)`, though on other objects than `OrtResult`. I propose to either make clear in the commit message that this targets only `OrtResult`, or replace the other calls with `toYaml()`, too.

Reply:
I provided the class name in the commit title to make the scope (this class) of the commit clear. This doesn't help?

Reply:
Yes, sorry, that's enough.

Reply:
Got used to commits which fix things across multiple tests I guess....no wonder why